app/src/main/AndroidManifest.xml

```diff
@@ -98,6 +98,10 @@
                 android:value="io.github.droidkaigi.confsched2018.presentation.MainActivity"
                 />
         </activity>
+        <meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="your API key"
```

Comment:
It seems necessary google map api key.
Is there it ?

https://developers.google.com/maps/documentation/android-api/signup?hl=ja#key

Reply:
Sorry, I ask konifar. Please wait 🙇

Reply:
Yes, it's necessary. @takahirom Can you create it? 🙏

Reply:
@atsuya046

Reply:
thanks!!